src/app/components/header/login/login.component.css

```diff
@@ -0,0 +1,36 @@
+.panel-heading {
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+#forget {
```

Comment:
do not use IDs to target elements in css

Reply:
okay, will keep that in mind.